integrations/src/test/java/io/jenkins/plugins/casc/GlobalToolGlobalMavenTest.java

```diff
@@ -1,43 +0,0 @@
-package io.jenkins.plugins.casc;
-
-import hudson.ExtensionList;
-import hudson.tasks.Maven;
-import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
-import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
-import jenkins.model.Jenkins;
-import jenkins.mvn.GlobalMavenConfig;
-import org.junit.Rule;
-import org.junit.Test;
-import org.jvnet.hudson.test.Issue;
-
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-
-public class GlobalToolGlobalMavenTest {
```

Comment:
I found this and consider it as a duplicate, and given its comments, decide to remove it